purchase_allowed_product/__manifest__.py

```diff
@@ -1,6 +1,7 @@
 # © 2016 Chafique DELLI @ Akretion
 # © 2017 Today Mourad EL HADJ MIMOUNE @ Akretion
 # 2020 Manuel Calero - Tecnativa
+# 2025 Robrecht Colson - Gumbys
```

Comment:
Please don't add yourself as an author.
You're welcome to add your name to the contributors list in the readme directory.